components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -25,6 +25,33 @@ class ActionSheetTest: XCTestCase {
     actionSheet = MDCActionSheetController()
   }
 
+  private func getLabel(with text: String) -> UILabel? {
```

Comment:
This test method makes me think that we should just expose the label.

Once we do that we can style the title and message in an external object. This allows us to follow [mutator design pattern](http://go/mutator-design-pattern)'s "dumb components." This is how we ended up with our themers.